corehq/apps/domain/templates/domain/select_plan.html

```diff
@@ -39,8 +39,35 @@
     to learn more</a> about each plan.
 {% endblocktrans %}
 </p>
+
+<div style="float: right;">
+    <div class="btn-group btn-group-toggle" id="toggle-annual-pricing" data-toggle="buttons">
+        <label class="btn btn-primary active" data-bind="click: updateIsAnnualPricing">
```

Comment:
I'm also a little confused about the markup here - are these meant to be buttons? If they are, use `button` elements instead of labels and don't use radio buttons at all...which would then mean the above advice about a `checked` binding wouldn't apply, because `checked` is only for checkboxes/radios...so then a `click` binding that calls a function to directly update `isAnnualPricing` would make sense.

Not sure if you have the two separate functions for readability reasons, but it's also fine to pass parameters to the function, so you could do something like `click: function() { updateIsAnnualPricing(true) }` (the function wrapper is to [prevent knockout from executing the function on page load](https://stackoverflow.com/questions/9891282/knockoutjs-bindings-with-parameters-triggered-on-load)).